examples/vision/video_classification.py

```diff
@@ -1,19 +1,25 @@
 """
-Title: Video Classification with a CNN-RNN Architecture
-Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Video Classification with a CNN-RNN Architecture
+Modified, corrected, and updated: [Mikolaj Buchwald](https://www.linkedin.com/in/mikolaj-buchwald/)
+Author of the original example: [Sayak Paul](https://twitter.com/RisingSayak)
```

Comment:
I don't think this is supported by the backend. You will have to keep those `Title:` `Author:` formats. 

But I am okay if you give yourself an acknowledgment at the end of the tutorial.

Reply:
Ok, I will correct that, I will mention you in the description/markdown below.

Reply:
Sorry, I meant that you will still be the "Author" and **I** will be mentioned in the acknowledgement, if that is all right.

Reply:
@sayakpaul , according to the README, it is possible to use the "Authors" tag (i.e., multiple authors).

Can I ask to become a co-author (a second author) of the example (with the PRs)?

See:

```
Author: (could be `Authors`: as well, and may contain markdown links)
```

https://github.com/keras-team/keras-io

Reply:
Hi, in the following commits I listed myself as a co-author, as there were quite a few changes, in order for this example to work. Thank you Sayak for your review and all the help, you are still the first, and the original author.

Please see my replies to the other issues you mentioned.